Testimonial model

miriam-gale opened the pull request:
closes #1

KingsleyNahyi committed:
Production issue solved.

miriam-gale committed:
backend

miriam-gale committed:
Testimonial model

miriam-gale committed:
Testimonial model completed closes #1

miriam-gale committed:
Testimonial model completed closes #1

FrancisSimple reviewed (changes requested):
What is that natural language file inside models folder

miriam-gale committed:
fix: Removed unintended NLP/ML file from models directory as per reviewer request.

miriam-gale commented:
@FrancisSimple 
I have pushed the fix for the requested change. I removed the file that was in the `src/models` directory. This commit addresses your feedback. Please can you review it?

miriam-gale committed:
Address remaining migration file concern. Forcing PR update.

miriam-gale committed:
final cleanup: Committed deleted migration files and finalized PR.

miriam-gale committed:
Updated Testimonial model
